app/views/layouts/_subnav.html.erb

```diff
@@ -1,8 +1,38 @@
-<% if content_for? :subnav %>
-  <nav class="navbar navbar-dark bg-secondary">
-    <div class="container">
-      <%= link_to 'Subnav', fake_index_path, class: 'navbar-brand' %>
-      <%= yield :subnav %>
+<nav class="prx-app-nav navbar bg-white d-flex flex-row flex-wrap justify-content-start m-0 p-0">
+
+    <!-- PODCAST SWITCHER -->
```

Comment:
Some more indent issues - I'll install an html/erb linter soon as well.  So this is fine!